packages/cli/src/ui/components/Footer.tsx

```diff
@@ -41,13 +42,17 @@ export const Footer: React.FC<FooterProps> = ({
   promptTokenCount,
   nightly,
   vimMode,
+  numRenders,
 }) => {
   const limit = tokenLimit(model);
   const percentage = promptTokenCount / limit;
 
   return (
     <Box justifyContent="space-between" width="100%">
       <Box>
+        {debugMode && numRenders > 0 && (
```

Comment:
I'm not sure all users of debug mode want to see this. Can we put it behind a setting or require you to press a keyboard shortcut to enable it. I'd suggest maybe F12 as that is the keyboard shortcut to typically bring up devtools.

Reply:
Done.